packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -693,16 +693,6 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {	// eslint-disable-line
 				this.calculateMaxComponentCount(leftCount, rightCount, childrenCount);
 			}
 
-			if (this.props.visible && !nextProps.visible) {
```

Comment:
I'm not sure about removing this. I couldn't see any place that `showMoreComponents` was reset to false beyond the imperative methods exposed by `ApiDecorator`. Can you elaborate on why this is safe?

Reply:
In testing this, it appears that if you click "more", click the video to hide the controls, and click again to show them, it shows the more controls rather than the playback. I'm assuming that's due to this change.

Reply:
Possible fix :)
```js
static getDerivedStateFromProps (props) {
	if (!props.visible) {
		return {
			showMoreComponents: false
		};
	}

	return null;
}
```